package/MDAnalysis/transformations/fit.py

```diff
@@ -174,8 +183,20 @@ class fit_rot_trans(object):
     Returns
     -------
     MDAnalysis.coordinates.base.Timestep
+
+
+    .. versionchanged:: 2.0.0
+       The transformation was changed from a function/closure to a class
+       with ``__call__``.
+    .. versionchanged:: 2.0.0
+       The transformation was changed to inherit from the base class for
+       limiting threads and checking if it can be used in parallel analysis.
     """
-    def __init__(self, ag, reference, plane=None, weights=None):
+    def __init__(self, ag, reference, plane=None, weights=None,
+                 max_threads=1, parallelizable=True):
```

Comment:
In some of the transformations `max_threads` is set to 1, whilst in the others it is set to `None`. My understanding is that the ones that are set to 1 are the cases where you've benchmarked the transformation and it looks like setting to 1 is faster. Would it make sense in these cases to add a `Note` to the docstring documenting this? That way next time folks come along and see the differences they can easily see why that design decision was made.